app/src/main/java/mozilla/org/webmaker/WebmakerApplication.java

```diff
@@ -30,9 +30,9 @@ public void onCreate() {
         Router.sharedRouter().setContext(getApplicationContext());
         Router.sharedRouter().map("/main", MainActivity.class);
         Router.sharedRouter().map("/main/:tab", MainActivity.class);
-        Router.sharedRouter().map("/map/:projectId", Map.class);
-        Router.sharedRouter().map("/projects/:projectId", Project.class);
-        Router.sharedRouter().map("/projects/:projectId/elements/:elementId", Editor.class);
+        Router.sharedRouter().map("/map/:projectId", Project.class);
```

Comment:
:+1: I was hoping to land #1667 before we did this though as it's going to make my patch really difficult to merge.